subsys/mpsl/mpsl_init.c

```diff
@@ -233,6 +226,19 @@ static int mpsl_signal_thread_init(const struct device *dev)
 	return 0;
 }
 
+#if IS_ENABLED(CONFIG_MPSL_FEM)
+static int mpsl_fem_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+	return mpsl_fem_configure();
+}
+#endif
+
 SYS_INIT(mpsl_lib_init, PRE_KERNEL_1, CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
 SYS_INIT(mpsl_signal_thread_init, POST_KERNEL,
 	 CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
+
+#if IS_ENABLED(CONFIG_MPSL_FEM)
+SYS_INIT(mpsl_fem_init, POST_KERNEL, CONFIG_MPSL_FEM_INIT_PRIO);
+#endif
```

Comment:
Wouldn't it be better to move it to `mpsl_fem.c` and avoid this `#if`?

Reply:
Sure.

Reply:
I've decided that there's no need to move this line after all. Let me know if you agree